A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/OPTIONS/A320_Neo_CDU_OPTIONS_AOC_SIMBRIEF.js

```diff
@@ -20,22 +20,18 @@ class CDU_OPTIONS_SIMBRIEF {
     static ShowPage(mcdu) {
         mcdu.clearDisplay();
 
-        const simbriefUsername = NXDataStore.get("CONFIG_SIMBRIEF_USERNAME", "").replace(/_/g, ' ');
+        const simbriefUser = NXDataStore.get("CONFIG_SIMBRIEF_USERID", "");
 
-        const simbriefUsernameString = simbriefUsername ? `{green}[${simbriefUsername}]{end}` : "{cyan}*[\xa0\xa0\xa0\xa0\xa0]{end}";
-
-        const simbriefUserId = NXDataStore.get("CONFIG_SIMBRIEF_USERID", "");
-
-        const simbriefUserIdString = simbriefUserId ? `{green}[${simbriefUserId}]{end}` : "{cyan}*[\xa0\xa0\xa0\xa0\xa0]{end}";
+        const simbriefUserString = simbriefUser ? `{green}[${simbriefUser}]{end}` : "{cyan}*[\xa0\xa0\xa0\xa0\xa0]{end}";
```

Comment:
If the simbiref user ID has a fixed length, I'd suggest using _ to display entry boxes.

Also I'd stick to the airbus colors, since it's pilot entered and therefore selected -> using color cyan
```suggestion
        const simbriefUserString = simbriefUser ? `{cyan}\xa0${simbriefUser}{end}` : "{amber}\xa0______{end}";
```

Reply:
Agree on the colour changes - fixed.

I think SimBrief user IDs can be shorter than 6 characters (I seem to remember helping people with 4-character user IDs before), and we also accept usernames here - so I'm not sure putting entry boxes makes sense.

Reply:
Well it's the same with company routes I guess, afaik I know, they have a maximum length, however now a minimum.

Reply:
Just another idea, we could add a HELP> Page to explain the user how to setup/"link" his simbrief account.

Reply:
It's fine with me either way.